in_toto/keylib.go

```diff
@@ -14,6 +14,29 @@ import (
   "reflect"
 )
 
+func ParseRSAPublicKeyFromPEM(pemBytes []byte) (*rsa.PublicKey, error) {
+  // TODO: There could be more key data in _, which we silently ignore here.
+  // Should we handle it / fail / say something about it?
```

Comment:
If we were to merge this, could we ticketize this issue?

Reply:
Yes. Done. #14.